sklearn/utils/mocking.py

```diff
@@ -55,7 +58,7 @@ def fit(self, X, y):
     def predict(self, T):
         if self.check_X is not None:
             assert_true(self.check_X(T))
-        return T.shape[0]
+        return np.zeros(_num_samples(T))
```

Comment:
It seems to be a classifiers. So it should be `self.classes_[np.zeros(_num_samples(T))]`.